src/components/ActivityCard.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-container pa-0>
+    <v-card>
+      <v-img
+        :height="imageHeight"
+        :src="imgSrc"
+      ></v-img>
+      <v-card-title class="no-padding-bottom">
+        <div class="text-truncate">
+          <p class="title text-truncate">
+            {{ title }}
+          </p>
+          <p class="body-2 grey--text text-truncate">
+            {{ description }}
+          </p>
+        </div>
+      </v-card-title>
+      <v-card-actions class="no-padding-top">
+        <v-btn flat>
+          Saber mais
+        </v-btn>
+        <v-spacer></v-spacer>
+        <v-btn :color="buttonColor">
+          {{ finalized ? 'Finalizada' : 'Feito' }}
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-container>
+</template>
+
+<script>
+
+export default {
+  name: 'activity-card',
+  props: {
+    description: {
+      type: String,
+      required: true,
+    },
+    imgSrc: {
+      type: String,
+      required: true,
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    finalized: Boolean,
+  },
+  computed: {
+    buttonColor() {
+      return this.finalized ? 'success' : 'secondary';
+    },
+    imageHeight() {
+      switch (this.$vuetify.breakpoint.name) {
+        case 'xs': return '150px';
+        case 'sm': return '200px';
+        case 'md': return '200px';
+        case 'lg': return '200px';
+        case 'xl': return '200px';
+        default: console.log('inválid screen size.');
+      }
+
+      return '';
```

Comment:
Using this, maybe we don't need a default case and the console.log could stay here OR we could return ' ' inside the code block in default case